src/pages/Flow/Details-Tile.vue

```diff
@@ -209,61 +297,255 @@ export default {
               </div>
             </v-list-item-content>
           </v-list-item>
+          <v-list-item dense class="px-0">
+            <v-list-item-content width="800px" class="overflow-x-scroll">
+              <v-list-item-subtitle class="caption">
+                Labels
+                <v-menu
+                  v-model="noLabelInfo"
+                  :close-on-content-click="false"
+                  offset-y
+                  open-on-hover
+                >
+                  <template v-slot:activator="{ on }">
+                    <v-btn
+                      :color="labels.length < 1 ? 'warning' : null"
+                      text
+                      icon
+                      x-small
+                      v-on="on"
+                    >
+                      <v-icon>
+                        info
+                      </v-icon>
+                    </v-btn>
+                  </template>
+                  <v-card tile class="pa-0" max-width="320">
+                    <v-card-title class="subtitle pb-1"
+                      >Flow group labels</v-card-title
+                    >
+
+                    <v-card-text class="pt-0">
+                      Flows and agents have optional labels which allow you to
+                      determine where your flows are executed. For more
+                      information see
+                      <a
+                        href="https://docs.prefect.io/orchestration/execution/overview.html#labels"
+                        target="_blank"
+                        >the docs on labels</a
+                      >.
+
+                      <v-alert
+                        v-if="labels.length < 1"
+                        border="left"
+                        colored-border
+                        type="warning"
```

Comment:
Could we relax the display of this message?  There's nothing wrong with having a flow with no labels, but the exclamation point makes me second guess that